database/migrations/2019_09_04_152016_create_deliveries_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateDeliveriesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('deliveries', function (Blueprint $table) {
+            $table->increments('id');
+            $table->integer('centre_id')->unsigned();
+            $table->string('range');
+            $table->timestamp('dispatched_at');
+            $table->timestamps();
+
+            $table->foreign('centre_id')
+                ->references('id')
+                ->on('centres');
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::dropIfExists('deliveries');
```

Comment:
Do you have to drop foreign key first?

Reply:
Nope, drop table is pretty sledgehammer :hammer: We'd have to drop foreigns that are dependent on this table, but thats more a table ordering thing.